server/src/main/java/com/nukkitx/server/level/biome/impl/PlainsBiomeImpl.java

```diff
@@ -0,0 +1,19 @@
+package com.nukkitx.server.level.biome.impl;
+
+import com.nukkitx.api.block.BlockTypes;
+import com.nukkitx.server.block.NukkitBlockState;
+import com.nukkitx.server.level.biome.BaseCoveredBiome;
+
+/**
+ * @author DaPorkchop_
+ */
+public class PlainsBiomeImpl extends BaseCoveredBiome {
+    public PlainsBiomeImpl() {
```

Comment:
why you don't do a space ? i really like to rewrite have full reformat and will be perfect.

Reply:
It's fine imo. You wouldn't leave a gap at the bottom where the braces end so why would you need to at the top.

Reply:
i actually intentionally delete that space at the beginning of classes whenever i see it, looks pretty ugly imo

Reply:
What about just letting an IDE reformat everything, with given formatting style and comit

Reply:
i DO that
my IDE removes that because i configure it to, but i won't let it reformat the whole project because that's out of the scope of this PR

Reply:
Then someone should do that because the formatting is a mess in Nukkit since everyone does it how they want to. Yes it works, but it does not seems nice imo.

Reply:
the rewrite branch has nearly decent formatting, as it was all made by @SupremeMortal in intellij. i'll probably make a PR soon tm that includes formatting presets for idea and eclipse that people have to use to have their PRs merged